app/src/main/java/crux/bphc/cms/helper/CourseDataHandler.java

```diff
@@ -100,6 +102,19 @@ public String getActionBarTitle(int courseId){
         return courses;
     }
 
+    public List<Course> rearrangeCourses(List<Course> courseList) {
```

Comment:
This function, in essence, deals with the final course list that is shown to the user. As such, it is a view method and not a data handler/controller method. Move to `MyCourseFragment.java`